Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/security/SecurityHelper.kt

```diff
@@ -23,26 +23,44 @@ import android.annotation.SuppressLint
 import android.content.SharedPreferences
 import android.os.Build
 import android.util.Base64
-import de.rki.coronawarnapp.CoronaWarnApplication
+import androidx.annotation.VisibleForTesting
 import de.rki.coronawarnapp.exception.CwaSecurityException
+import de.rki.coronawarnapp.util.di.AppInjector
+import de.rki.coronawarnapp.util.di.ApplicationComponent
 import de.rki.coronawarnapp.util.security.SecurityConstants.CWA_APP_SQLITE_DB_PW
 import de.rki.coronawarnapp.util.security.SecurityConstants.DB_PASSWORD_MAX_LENGTH
 import de.rki.coronawarnapp.util.security.SecurityConstants.DB_PASSWORD_MIN_LENGTH
 import de.rki.coronawarnapp.util.security.SecurityConstants.ENCRYPTED_SHARED_PREFERENCES_FILE
+import timber.log.Timber
 import java.security.SecureRandom
 
 /**
  * Key Store and Password Access
  */
 object SecurityHelper {
```

Comment:
this should be an injectable class, ideally with interface (maybe out of scope for this PR -> 1 more for refactoring epic)

Reply:
Yep out of scope here, would require too many changes, but should be refactored as part of our no-more-singletons tickets.